src/simulation_lifecycle/simulation.c

```diff
@@ -89,3 +92,58 @@ int write_sim_config(const cJSON *simulation_config, char *config_json) {
     }
     return write_data_to_file(cJSON_GetStringValue(config_output_path), config_json);
 }
+
+int run_sim(const cJSON *simulation_config){
+
+    /* Get model name - e.g. 2_2_agent_sir_config */
+    cJSON *model_id = cJSON_GetObjectItemCaseSensitive(simulation_config, SIM_MODEL_ID);
+    char *model = cJSON_GetStringValue(model_id);
+
+    /* Define executable - e.g. 2_2_agent_sir_config.exe */
+    char *model_executable = concat(model, ".exe");
+
+    /* Get config output path */
+    cJSON *config = cJSON_GetObjectItemCaseSensitive(simulation_config, SIM_CONFIG_OUTPUT_PATH);
+    char *config_path = cJSON_GetStringValue(config);
+
+    /* Define a command and run simulation:
+     * The command is changing to the directory containing the .exe file.
+     * Then, the command executes the .exe file with one argument
+     * which is the scenario config json file.
+     * Example: cd ../third_party/CellDEVS_models/tutorial/bin/ ;
+     * ./2_2_agent_sir_config.exe ../../../../test/data/run_simulation/config.json
+     */
+    char *command = concat("cd ",SIM_MODEL_LIBRARY);
```

Comment:
look at `sprintf` function. It is more readable and user friendly than doing multiple concatenations ;)
https://stackoverflow.com/questions/804288/creating-c-formatted-strings-not-printing-them